prometheus/base/alerting/prometheus.pod_alerts.yaml

```
@@ -25,3 +25,14 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is crash looping"
         description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') for more than 15 minutes."
+    - alert: QuotaExceeded
+      expr: |
+        kube_resourcequota{type="used", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} /
+        ignoring (instance, job, type)
```

@amisstea do you recall what was the idea behind ignoring instance and job?

---

I'm not quite sure. I don't see `instance` as an available label on this metric and `job` is always the same value from all samples I've seen. Perhaps the implementers of the platform rule found some edge cases where these must be ignored. I suggest we ignore only `type` for now and revisit if it becomes a problem. Let's not carry forward potential tech debt we can't explain.

---

@ShakedAviv50313 can you adjust it accordingly?
```suggestion
        ignoring (type)
```

---

Done